checker/tests/initialization/EISOPIssue363a.java

```diff
@@ -0,0 +1,92 @@
+// Test case for issue 363:
+// https://github.com/eisop/checker-framework/issues/363
+
+import org.checkerframework.checker.initialization.qual.Initialized;
+import org.checkerframework.checker.initialization.qual.NotOnlyInitialized;
+import org.checkerframework.checker.initialization.qual.UnknownInitialization;
+
+class MyException extends Exception {
```

Comment:
Try having just one top-level class in tests or include something, like `363a` in all top-level class names.
Otherwise, it is very easy for two different tests to define a generic class name like `MyException` and strange behavior can result.